hail/python/dev/requirements.txt

```diff
@@ -1,3 +1,5 @@
+-c ../requirements.txt
```

Comment:
Can you double check this is right?

Reply:
Yup, this will "constrain" the requirements in this file with those in the upper requirements.txt, so we know the two requirements files won't diverge/conflict.

Reply:
I double-checked by pip installing the hail dependencies and dev dependencies